packages/web/src/components/gcds-button/gcds-button.tsx

```diff
@@ -114,6 +114,12 @@ export class GcdsButton {
    */
   @Prop() download: string | undefined;
 
+  /**
+   * Set to true if the button links to an external page.
+   * Requires the "type" to be set to "link" to work.
+   */
+  @Prop() external?: boolean = false;
```

Comment:
Is it possible to separate the new prop from this PR so we don't introduce a change to this component for now? We can add a prop in a separate PR and go through a component update process.